lib/EnsEMBL/REST/Controller/Info.pm

```diff
@@ -189,6 +189,9 @@ sub biotype_group : Path("biotypes/groups") Args(1) ActionClass('REST') {
     $c->go('ReturnError', 'custom', [qq{$_}]);
   };
 
+  ## Return 404 for get requests with no return
+  $c->go( 'ReturnError', 'not_found', ["biotypes not found for group $group"]) unless $biotypes->[0];
```

Comment:
Does this produce unitialised warnings when it triggers? It would probably be less warny to check the list size.